app.rb

```diff
@@ -31,8 +31,8 @@
   @payload = JSON.parse(payload_body)
 
   # If $required_plus_ones has been specified, change the default
-  if params['required_plus_ones']
-    NEEDED_PLUS_ONES = params['required_plus_ones']
+  if params['required_plus_ones'].to_i > 0
```

Comment:
I think if a client passes in `null` as a `POST` param or if it doesn't pass in the parameter at all, we should ignore it completely.